2020/webkitgtk-npapi-sunsetting.markdown

```diff
@@ -0,0 +1,195 @@
+title: Sunsetting NPAPI plug-in support in WebKitGTK (and WPE)
+date: 2020-09-28 12:00:00
+author: aperez
+tags: igalia, webkit, floss
+
+<ol class="toc">
+  <li><a href="#summary">Summary</a></li>
+  <li><a href="#what-is-npapi">What is NPAPI?</a></li>
+  <li><a href="#what-is-npapi-used-for">What is NPAPI used for?</a></li>
+  <li><a href="#why-are-npapi-plug-ins-being-phased-out">Why are NPAPI plug-ins being phased out?</a></li>
+  <li><a href="#what-are-other-browsers-doing">What are other browsers doing?</a></li>
+  <li><a href="#is-webkitgtk-following-suit">Is WebKitGTK following suit?</a></li>
+</ol>
+
+## Summary
+
+Here's a [tl;dr](https://www.urbandictionary.com/define.php?term=tl%3Bdr)
+list of bullet points:
+
+* One year ago, [WebKitGTK 2.26.0](https://www.webkitgtk.org/2019/09/09/webkitgtk2.26.0-released.html)
+  [removed
+  support](/2019/webkit-2.26-six-month-retrospective.html#cleanups) for NPAPI
+  plug-ins which used GTK2, but the rest of plug-ins kept working.
+* WebKitGTK 2.30.x will be the last stable series with support for NPAPI
+  plug-ins _at all_. Version [2.30.0 was
+  released](https://webkitgtk.org/2020/09/11/webkitgtk2.30.0-released.html) a
+  couple of weeks ago.
+* WebKitGTK 2.32.0, due in March 2021, will be the first *stable* release
+  to ship without support for NPAPI plug-ins.
+* We have already [removed the relevant
+  code](https://trac.webkit.org/changeset/265753/) from the WebKit repository.
+* While the WPE WebKit port allowed running windowless NPAPI plug-ins, this
+  was never advertised nor supported by us.
+
+
+## What is NPAPI?
+
+NPAPI, short for _Netscape Plugin Application Programming Interface_ allowed
+browser plug-ins to extend the functionality of a web browser providing
+support for additional content types. NPAPI appeared first in Netscape
+Navigator 2.0, in 1995.
+
+When a NPAPI plug-in is used to render content, the web browser will
+effectively _carve a hole_ in the rectangular location where content handled
```

Comment:
Remove *will effectively*, and change *carve* to *carves*. I see no purpose to use future tense. Keep it simple.

I also suggest the removal of *effectively* as it makes the sense wordy to my eyes.